### DISCUSSION TITLE
Sprint 6 meeting | 2022-11-29

### DISCUSSION THREAD

**Discussion by HuangJunye** [QPong/planning#37] [Meetings]
## Agenda

- [x] Sprint review (10 min)
- [x] Sprint retro (10 min)
- [ ] Sprint planning (10 min)
- [ ] Q&A (5 min)
- [ ] Game dev resources and news (5 min)

**Comment by HuangJunye** [comment]
## Review

[Last sprint planning](https://github.com/QPong/planning/discussions/35#discussioncomment-4157824)

### Dayeong

### Unathi

### Siddharth

### Junye 

### Follow up

**Comment by HuangJunye** [comment]
## Retro

### What went well


### What didn't go well


### What to improve?

**Reply by Unathi-Skosana** [reply]
## What went well?

- I made a buggy two-player MVP with rooms
- Fixed the low precision bugs
- 

## What didn't go well?

- N/A

## What to improve?

-  Fixing the rooms bugs and deploying to heroku

**Comment by HuangJunye** [comment]
## Planning

### Dayeong
- work on final presentation and demo

### Unathi
- fix the bugs of MVP
- deploy server on the cloud

### Siddharth


### Junye
- work on final presentation and demo
